src/mmframe.cpp

```diff
@@ -1072,6 +1073,30 @@ void mmGUIFrame::OnPopupReallocateAccount(wxCommandEvent& WXUNUSED(event))
         ReallocateAccount(account_id);
     }
 }
+
+void mmGUIFrame::OnPopupAccountBaseBalance(wxCommandEvent& WXUNUSED(event))
+{
+    if (selectedItemData_)
+    {
+        int account_id = selectedItemData_->getData();
+        Model_Account::Data* account = Model_Account::instance().get(account_id);
+        Model_Currency::Data* acc_currency = Model_Account::currency(account);
+        Model_Currency::Data* base_currency = Model_Currency::GetBaseCurrency();
+
+        double acc_bal = Model_Account::balance(account);
+        double acc_base_bal = acc_bal * acc_currency->BASECONVRATE;
+
+        wxString message = wxString::Format(
+            wxTRANSLATE("Account: %s\n\n"
+                        "Balance at currency %s: %s\n"
+                        "Balance at currency %s: %s"),
+            account->ACCOUNTNAME,
+            acc_currency->CURRENCY_SYMBOL, Model_Currency::toCurrency(acc_bal, acc_currency),
+            base_currency->CURRENCY_SYMBOL, Model_Currency::toCurrency(acc_base_bal));
+        wxMessageBox(message, wxTRANSLATE("Foreign Currency Account Balance"));
```

Comment:
Why 
~~~~~~
wxTRANSLATE("Foreign Currency Account Balance")
~~~~~~
should be 
_("Foreign Currency Account Balance")